source/synthSettingsRing.py

```diff
@@ -4,7 +4,6 @@
 import queueHandler
 from autoSettingsUtils.driverSetting import BooleanDriverSetting, NumericDriverSetting
 
-

```

Comment:
can you please restore this new line? otherwise this is ready to go

Reply:
Done👍🏻